CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+v4.2.0
+======
+- an updated version of Facebook SDK for Android, which should enable publishing apps with this module again
+- fixes issues with `react-native link` on Android
```

Comment:
Should v4.2.0 be the newest? I wouldn't mention `react-native link` anywhere anymore, that command has been deprecated a long time ago (react-native 0.60)

Reply:
I think the point here is to have a historical changelog for now. We could definitely add `## Unreleased` section with what we have on master

Reply:
Well the stuff we have on master should be released asap anyways, since the current package does not work at all (facebook has shut SDK v5 down)